native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,197 @@
+                                    node.identifier().toString().trim(),
+                                    defaultValues[indexOf(nodeMap,
+                                            node.identifier().toString().trim())], null));
+                        }
+                    }
+                } else {
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            } else if (member instanceof TableNode || member instanceof TableArrayNode) {
+                if (!existingNodes.isEmpty() || !moduleMembers.isEmpty()) {
+                    moduleMembers = addNewLine(moduleMembers, 2);
+                    moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+                    moduleMembers = addNewLine(moduleMembers, 1);
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                } else {
+                    isTableEntry = true;
+                    moduleMembers = moduleMembers.add((DocumentMemberDeclarationNode) member);
+                }
+            }
+        }
+        if (existingNodes.isEmpty() && isTableEntry == true) {
+            moduleMembers = addNewLine(moduleMembers, 2);
+            moduleMembers = moduleMembers.add(SampleNodeGenerator.createTable(name, null));
+            moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+            moduleMembers = addNewLine(moduleMembers, 1);
+            
+        } else if (existingNodes.size() != 5) {
+            moduleMembers = addNewLine(moduleMembers, 2);
+            moduleMembers = poulateRemaining(moduleMembers, existingNodes);
+        }
+
+        Token eofToken = AbstractNodeFactory.createIdentifierToken("");
+        DocumentNode documentNode = NodeFactory.createDocumentNode(moduleMembers, eofToken);
+        TextDocument textDocument = TextDocuments.from(documentNode.toSourceCode());
+        SyntaxTree syntaxTreeFinal = SyntaxTree.from(textDocument);
+        return syntaxTreeFinal;
+    }
+
+
+    private static boolean isDatabaseConfigurationEntry(KeyNode key) {
+        switch (key.toString().trim()) {
+            case "user":
+            case "database":
+            case "password":
+            case "host":
+            case "port":
+                return true;
+            default:
+                return false;
+        }
+    }
+    private static int indexOf(String[] arr, String key) {
+        int index = 0;
+        for (Object member : arr) {
+            if (key.equals((String) member)) {
+                return index;
+            }
+            index += 1;
+        }
+        return -1;
+    }
+
+    private static NodeList populateNodeList(NodeList moduleMembers, boolean create) {
+        moduleMembers = moduleMembers.add(SampleNodeGenerator.createStringKV("host", DEFAULT_HOST, null));
```

Comment:
Move keys (host, port ...) to constants.

Reply:
Updated in [e4140c3](https://github.com/ballerina-platform/persist-tools/pull/9/commits/e4140c3eaed66b42da875af1efb0cf2358ae3f01)